code/game/objects/items/spear.dm

```diff
@@ -181,8 +181,9 @@
 	desc = "A haphazardly-constructed yet still deadly weapon. The pinnacle of modern technology."
 	force = 12
 	throwforce = 22
-	armour_penetration = 15 //Enhanced armor piercing
+	reach = 2 //SKYRAT EDIT
```

Comment:
you added back the bone spear's reach again but didnt mention it in the PR body

Reply:
Oh damn yeah.